### ISSUE TITLE
Making build log terminal scroll to the bottom 

### ISSUE THREAD

**Issue by TueeNguyen** [Seneca-CDOT/telescope#2815]
**What would you like to be added**:

Whenever there's a build happening, the build log does update but it doesn't scroll down to the bottom to display the latest changes.
We can investigate this function 
https://github.com/Seneca-CDOT/telescope/blob/63f41451f07def7eca20521ea728d1c2ef5d81be/src/api/status/public/js/build-log/check-for-build.js#L31-L44

**Why would you like this to be added**:

For better experience!

**Comment by DukeManh** [comment]
Hey, I think this can be achieved with CSS `column-reverse` to treat the flow of the content in a reversed direction.

**Comment by TueeNguyen** [comment]
Thanks, tho you didn't let me figure it out : D

**Comment by DukeManh** [comment]
Oops, I was steering you in the CSS direction vs JS.

**Comment by JerryHue** [comment]
I am wondering, is it possible to use `flex` on this? The `xterm.js` uses canvas to draw the text, so I am not really sure if CSS can be applied here.

I think we would have to interface with the [`xterm.js` API](https://github.com/xtermjs/xterm.js/blob/master/typings/xterm.d.ts) to be able to scroll to the latest changes.

**Comment by DukeManh** [comment]
That's true xterm uses an HTML canvas and draws text to it, so CSS won't help here.

**Comment by TueeNguyen** [comment]
> I am wondering, is it possible to use `column-reverse` on this? The `xterm.js` uses canvas to draw the text, so I am not really sure if CSS can be applied here.
> 
> I think we would have to interface with the [`xterm.js` API](https://github.com/xtermjs/xterm.js/blob/master/typings/xterm.d.ts) to be able to scroll to the latest changes.

Thanks for the suggestion
